deploy-deps.sh

```diff
@@ -38,6 +39,18 @@ check_req(){
     fi
 }
 
+install_argocd() {
```

Comment:
This should not run when using the EKS cluster as Terraform takes care of deploying ArgoCD.